### PR TITLE
events page created

### PR DESCRIPTION
**Pull request by atiilla**
1. Events page created
> - Upcoming events
> - Past Events

2. Event detail page

### CODE REVIEW

File: package-lock.json
Comment (ozantekin) [inline package-lock.json:1]: No need to add the lock file @atiilla

File: app/events/[id]/page.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+import Image from 'next/image';
+import { useParams } from 'next/navigation'
+import { useEffect, useState } from 'react';
+
+
+const EventDetailPage = () => {
+    const { id } = useParams();
+    const [event, setEvent] = useState(null);
+
+    const getEvent = (id) => {
```
Comment (ozantekin) [inline app/events/[id]/page.tsx:13]: Service operations related to 'kommunity' need to be moved to the 'lib' folder @atiilla

**Comment by ozantekin** [comment]
I’m merging this so I can make improvements on it. Thanks for your work!  🚀

**Comment by umutbasal** [comment]
https://github.com/HappyHackingSpace/landing-frontend-app/issues/29

we have opened an issue on this with some ideas for SEO.
it's ok to merge if url schema won't change

**Pull request merged**